scripts/mlmodel_tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+test_dir=$PWD/peekingduck
+echo "Testing code in: $test_dir"
+
+if ! (coverage run --source="$test_dir" -m pytest -m "mlmodel"); then
+    coverage report
+    echo "UNIT TESTING FAILED."
```

Comment:
Will be good for the 3 echo statements here to specify that it's testing only ml models

Reply:
Yes, agreed.